sound/soc/sof/ipc4-fw-reg.h

```diff
@@ -0,0 +1,125 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause) */
+/*
+ * This file is provided under a dual BSD/GPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ */
+
+#ifndef __IPC4_FW_REG_H__
+#define __IPC4_FW_REG_H__
+
+#define SOF_IPC4_INVALID_STREAM_POSITION	ULLONG_MAX
+
+#define SOF_IPC4_INVALID_NODE_ID	0xFFFFFFFF
```

Comment:
This is already defined in https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/sof/ipc4-topology.h#L56. Can fix it with a fixup.

Reply:
remove it

Reply:
it is still there?

Reply:
unreasonable , it is not in my local git repo.